### ISSUE TITLE
Media Page/Press Kit

### ISSUE THREAD

**Issue by RowdyChildren** [CodeForCharlotte/laddrdocs#8]
We need to add a Press Kit and make the media page a little better

**Comment by Jillzey** [comment]
Check out some other online press kits (i.e., MailChimp)

Included: 
Logo
Mission statement
pictures
Links
How to get involved

**Comment by Jillzey** [comment]
I also think maybe we set up a different page for the press kit. On the media page we list the press that we get (interviews, articles, that kind of thing) and then at the top there is a link to our press kit. What do you think?

**Comment by RowdyChildren** [comment]
I think that is better!

**Comment by scottelundgren** [comment]
I agree that is a press kit would be super helpful. I'll try to bring a volunteer on Thursday to work on this issue.
